.devcontainer/scipy2023/jupyter_lab_config.py

```diff
@@ -0,0 +1,7 @@
+# Configuration file for lab.
+
+c = get_config()  # noqa
+
+## The default URL to redirect to from `/`
+#  Default: '/lab'
+c.LabApp.default_url = '/lab/tree/workshops/scipy2023/index.ipynb'
```

Comment:
Perhaps related to the 403 errors I'm seeing trying to execute the notebook in jupyterlab, I did a quick look around and see https://github.com/orgs/community/discussions/55146 (there are a fair number of unable to run jupyterlab discussions so it might be too early for this integration with codespaces... seems best for VScode !)

Reply:
https://github.com/devcontainers/features/blob/d934503a050ba84e6b42a006aacd891c4088eb62/src/python/install.sh#L467-L472
```
c.ServerApp.allow_origin = '*'
c.NotebookApp.allow_origin = '*'
```
not sure though, this sort of networking stuff is confusing...

Reply:
I didn't even see your message yet, yea looks like something with CORS. Just put a config for that.

Reply:
> Perhaps related to the 403 errors I'm seeing trying to execute the notebook in jupyterlab, I did a quick look around and see https://github.com/orgs/community/discussions/55146 

I saw that error, when I tried it using the actual github jupyter lab editor. The way I'm doing it is actually proxying from the vscode. I think the allow all origins should fix it.